### PR TITLE
Have RedisBroadcast return a BroadcastException

### PR DESCRIPTION
**Pull request by xiCO2k**
Since there is a BroadcastException already used on the PusherBroadcast class, makes sense that RedisBroadcast also throw the same exception.

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
This is a breaking change.

File: src/Illuminate/Broadcasting/Broadcasters/RedisBroadcaster.php [hunk 1]
```diff
@@ -105,6 +108,8 @@ public function validAuthenticationResponse($request, $result)
      * @param  string  $event
      * @param  array  $payload
      * @return void
+     *
+     * @throws \Illuminate\Broadcasting\BroadcastException
```
Comment (GrahamCampbell) [inline src/Illuminate/Broadcasting/Broadcasters/RedisBroadcaster.php:112]: can't add throws doc here unless it also exists on the contract

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: src/Illuminate/Broadcasting/Broadcasters/RedisBroadcaster.php (hunk 1 above)
Comment (xiCO2k) [reply src/Illuminate/Broadcasting/Broadcasters/RedisBroadcaster.php:112]: Used the same logic, there is on the PusherBroadcaster.

Should I add it to the contract?

Comment (xiCO2k) [reply src/Illuminate/Broadcasting/Broadcasters/RedisBroadcaster.php:112]: Just added. makes sense.

**Pull request closed without merging**